specification/source/glossary.rst

```diff
@@ -0,0 +1,26 @@
+.. _sec_glossary:
+
+Glossary of Terms
+----------------------
+
+This section provides definitions for abbreviations and terms introduced in OBC.
+
+
+.. glossary::
+
+   OBC
+      Open Building Controls (OBC) is an open source building controls software under development at LBNL.
+
+   CDL
+      Control Description Language (CDL) is a graphical programming language based on Modelica used to program control logic in OBC.
+
+   Controls Design Tool
+      Controls Design Tool is an implementation of OBC software in OpenStudio. The tool can be used to design, generate and export a control sequence.
+
+   Functional Verification Tool
+      fixme: should satisfy requrements arising from any of the validation use cases

```

Comment:
The Functional Verification Tool is a software that takes as an input the control sequence in CDL, requirements expressed in CDL, a list of I/O connections, and a configuration file, and then tests whether the measured control signals satisfy the requirements, violate them, or whether some requirements remain untested.